apps/condo/pages/ticket/index.tsx

```diff
@@ -53,14 +52,87 @@ const TAP_BAR_ROW_GUTTER: [Gutter, Gutter] = [0, 20]
 const CHECKBOX_STYLE: CSSProperties = { paddingLeft: '0px', fontSize: fontSizes.content }
 const TOP_BAR_FIRST_COLUMN_GUTTER: [Gutter, Gutter] = [40, 20]
 
+const TICKETS_RE_FETCH_INTERVAL = 60 * 1000
+
+const TicketsTable = ({ filterMetas, sortBy, searchTicketsQuery }) => {
+    const router = useRouter()
+
+    const { filters, offset } = parseQuery(router.query)
+    const currentPageIndex = getPageIndexFromOffset(offset, DEFAULT_PAGE_SIZE)
+
+    const {
+        loading: isTicketsFetching,
+        count: total,
+        objs: tickets,
+        refetch,
+    } = Ticket.useObjects({
+        sortBy,
+        where: searchTicketsQuery,
+        first: DEFAULT_PAGE_SIZE,
+        skip: (currentPageIndex - 1) * DEFAULT_PAGE_SIZE,
+    }, {
+        fetchPolicy: 'network-only',
+    })
+
+    const [isRefetching, setIsRefetching] = useState(false)
+    useEffect(() => {
+        const handler = setInterval(async () => {
```

Comment:
You need to add a reloadable feature flag here!

Reply:
done